doc/glossary.rst

```diff
@@ -373,6 +373,28 @@
                 provides a gateway to both the Amazon S3 RESTful API and the
                 OpenStack Swift API. 
 
+        scrubbing
+
+                The process by which Ceph ensures data integrity. During the
+                process of scrubbing, Ceph generates a catalog of all objects
+                in a placement group, then ensures that none of the objects are
+                missing or mismatched by comparing each primary object against
+                its replicas, which are stored across other OSDs. Any OSD that
+                is determined to have a copy of an object that is different
+                than the other copies or is missing entirely is marked
+                "inconsistent" (that is, the OSD is marked "inconsistent"). 
+
+                There are two kinds of scrubbing: light scrubbing and deep
+                scrubbing (also called "normal scrubbing" and "deep scrubbing",
```

Comment:
First time I've ever heard of normal scrubbing.  I would remove the parenthetical.

Reply:
It's how "Mastering Ceph" refers to it. I'm inclined to keep it in, since "Mastering Ceph" is a decent synoptic look at Ceph. Even if (as I sense) "light scrubbing" is the more correct term, I want to link the different terms so that there is no confusion.

Reply:
"that is determined"